cargo-shuttle/src/main.rs

```diff
@@ -179,4 +182,28 @@ impl Shuttle {
         let owned = locks.get(0).unwrap().file().try_clone()?;
         Ok(owned)
     }
+
+    fn run_tests(&self, no_test: bool) -> Result<()> {
+        if no_test {
+            return Ok(());
+        }
+
+        let config = cargo::util::config::Config::default()?;
+        let working_directory = self.ctx.working_directory();
+        let path = working_directory.join("Cargo.toml");
+
+        let compile_options = CompileOptions::new(&config, CompileMode::Test).unwrap();
+        let ws = Workspace::new(&path, &config)?;
+        let opts = TestOptions {
+            compile_opts: compile_options,
+            no_run: false,
+            no_fail_fast: false,
```

Comment:
```suggestion
            ..Default::default()
```
Might fail if `TestOptions` does not implement `Default`

Reply:
Thanks for the feedback. What is `Default()` saving against here, any future changes to TestOptions in Cargo?  Adding the above code gives me the error "the trait `std::default::Default` is not implemented for `TestOptions`". Happy to do what's needed to get this to work if this is still needed, just point me in the right direction 😄

Reply:
Yip, saving against future changes and not having to worry about setting fields which are not needed. Since it does not implement `Default` this is all good